### ISSUE TITLE
Amount of stations to take is wrong

### ISSUE THREAD

**Issue by Babyelefant1312** [grote/Transportr#809]
**Describe the bug**
If you search for a connection betqeen two stations, it will show you always one Station less than actually.

**To Reproduce**
1. Station u osloerstr. Berlin to s Westhafen berlin
2. it will show 3 stations till arrival, evejthough its  @4
![Screenshot_20220309-160311_Transportr](https://user-images.githubusercontent.com/101272146/157468586-4246cda5-e74f-41c4-b7d9-bb4631af474a.png)

**Versions (please complete the following information):**
 - **Transportr Version 2.1.4.
 - **Device:** redmi note 9 pro
 - **Android 11

**Comment by mimi89999** [comment]
I think that it's correct. There is an expandable list of stations between stations. It's 3 stations between departure station and arrival station.

**Comment by Babyelefant1312** [comment]
Yes thats true, its possible to see it that way. But due to every other navigation systems it would be less confusing. I mean check on g maps for example. It's different.
Cheerio

05.04.2022 19:10:09 Michel Le Bihan ***@***.***>:

**Comment by ialokim** [comment]
Interesting, never noticed that difference between Transportr and other apps. Perhaps rephrasing it like "*passing* 3 stops" could help making it clearer. Although I'm not sure if that would lead to translation issues (e.g. in German I'm not sure how to express that shortly).

**Comment by Altonss** [comment]
Maybe the simplest way would be to increment it by 1?

**Comment by ialokim** [comment]
Maybe have a short review of other navigation apps, if its true that all of them handle it differently, I'd agree to change it in Transportr as well.

**Comment by Altonss** [comment]
Apps (or websites) that handle it differently:
- GoogleMaps
- bvg.de
- ratp.fr

**Comment by M00NJ** [comment]
I also think incrementing the stop count by 1 is the best solution. All the public transport apps I've used so far count stops this way, which caused some confusion when I switches to Transportr.

**Comment by 4jNsY6fCVqZv** [comment]
For the same connection Öffi prints: "3 intermediate stops".

**Comment by M00NJ** [comment]
That works too, but the string would be quite long then. Some translations might not even fit in one line. Here is german: "3 dazwischenliegende Haltestellen". Apart from that, I still think '4 stops' is easier to comprehend than '3 intermediate stops'. Also imo it makes sense for a public transport app to go for the simplest and most concise option to display information, because you might be in a hurry after all.

**Comment by 4jNsY6fCVqZv** [comment]
For me, the string in German would be "3 Zwischenstopps".

**Comment by 4jNsY6fCVqZv** [comment]
![1](https://github.com/grote/Transportr/assets/46536067/4088e54f-c8c7-40b0-bbe1-e8c0b09d743f)

When I look at my screenshot and the original screenshot, it makes more sense to me that there are "3 intermediate stops".

The arrows next to "3 Haltestellen" and "3 stops" show me that I can tap to reveal the stops that are between my selected start and destination.

The intermediate stops are each displayed in a contrasting grey colour. Just like "3 stops". For me, this is also an indication that the naming should be about the "in-between".

**Comment by 4jNsY6fCVqZv** [comment]
What irritates me, but would probably be an issue of its own, is that the start arrow of the route does not point towards the destination, but even away from it.
